sqrl-testing/sqrl-flink-1.18/src/test/java/com/datasqrl/SnowflakeIT.java

```diff
@@ -0,0 +1,90 @@
+
+    datasqrlRun.run(false);
+
+    Thread.sleep(1000);
+    Path schema = directoryPath.resolve("build/deploy/snowflake/database-schema.sql");
+    String[] statements = Files.readString(schema).split(";");
+    for (String statement : statements) {
+      statement = Strings.trim(statement);
+
+      if (Strings.isEmpty(statement)) continue;
+      try (Connection connection = DriverManager.getConnection(
+          datasqrlRun.getSnowflakeUrl().get())) {
+        connection.createStatement().execute(statement);
+      }
+    }
+
+    String graphqlEndpoint = "http://localhost:8888/graphql";
+    String query = "query {\n"
+        + "  SnowflakeQuery(customerId:1) {\n"
+        + "    customer_id\n"
+        + "    amount\n"
+        + "  }\n"
+        + "  MyIcebergTable(limit:1) {\n"
+        + "    customer_id\n"
+        + "  }\n"
+        + "}";
+
+    String response = executeQuery(graphqlEndpoint, query);
+
+
+    assertEquals(response, "{\"data\":{\"SnowflakeQuery\":[{\"customer_id\":1.0,\"amount\":706721.71},{\"customer_id\":1.0,\"amount\":706721.71},{\"customer_id\":1.0,\"amount\":706721.71},{\"customer_id\":1.0,\"amount\":42079.16},{\"customer_id\":1.0,\"amount\":42079.16},{\"customer_id\":1.0,\"amount\":42079.16},{\"customer_id\":1.0,\"amount\":463121.85},{\"customer_id\":1.0,\"amount\":463121.85},{\"customer_id\":1.0,\"amount\":463121.85},{\"customer_id\":1.0,\"amount\":290005.57},{\"customer_id\":1.0,\"amount\":290005.57},{\"customer_id\":1.0,\"amount\":290005.57},{\"customer_id\":1.0,\"amount\":253812.11},{\"customer_id\":1.0,\"amount\":253812.11},{\"customer_id\":1.0,\"amount\":253812.11},{\"customer_id\":1.0,\"amount\":29341.72},{\"customer_id\":1.0,\"amount\":29341.72},{\"customer_id\":1.0,\"amount\":29341.72},{\"customer_id\":1.0,\"amount\":243720.97},{\"customer_id\":1.0,\"amount\":243720.97},{\"customer_id\":1.0,\"amount\":243720.97},{\"customer_id\":1.0,\"amount\":16778.48},{\"customer_id\":1.0,\"amount\":16778.48},{\"customer_id\":1.0,\"amount\":16778.48},{\"customer_id\":1.0,\"amount\":255874.79},{\"customer_id\":1.0,\"amount\":255874.79},{\"customer_id\":1.0,\"amount\":255874.79},{\"customer_id\":1.0,\"amount\":605384.78},{\"customer_id\":1.0,\"amount\":605384.78},{\"customer_id\":1.0,\"amount\":605384.78},{\"customer_id\":1.0,\"amount\":379228.36},{\"customer_id\":1.0,\"amount\":379228.36},{\"customer_id\":1.0,\"amount\":379228.36},{\"customer_id\":1.0,\"amount\":8355.88},{\"customer_id\":1.0,\"amount\":8355.88},{\"customer_id\":1.0,\"amount\":8355.88},{\"customer_id\":1.0,\"amount\":39127.77},{\"customer_id\":1.0,\"amount\":39127.77},{\"customer_id\":1.0,\"amount\":39127.77},{\"customer_id\":1.0,\"amount\":252694.58},{\"customer_id\":1.0,\"amount\":252694.58},{\"customer_id\":1.0,\"amount\":252694.58},{\"customer_id\":1.0,\"amount\":214574.98},{\"customer_id\":1.0,\"amount\":214574.98},{\"customer_id\":1.0,\"amount\":214574.98}],\"MyIcebergTable\":[{\"customer_id\":4.0}]}}\n");
```

Comment:
i think this is incorrect, there should be only one set but there are 3 here. Its because i reran the population script and it appended more records